src/utils/throwable-to-error.ts

```diff
@@ -0,0 +1,11 @@
+export default function throwableToError(e: unknown): Error {
+  if (e instanceof Error) {
+    return e;
+  }
+
+  const error = new Error(typeof e === 'object' ? `${JSON.stringify(e)}` : `${String(e)}`);
```

Comment:
Can we simplify this to:

const error = new Error(typeof e === 'object' ? JSON.stringify(e) : '' + e);

Reply:
done 👍